packages/astro/test/units/assets/fonts/utils.test.js

```diff
@@ -105,7 +116,8 @@ describe('fonts utils', () => {
 		}
 	});
 
-	it('createCache()', async () => {
+	it('cache()', async () => {
+		// eslint-disable-next-line @typescript-eslint/no-shadow
 		const { cache, getKeys } = createSpyCache();
```

Comment:
I strongly suggest listening to the linter when it comes to `no-shadow`. I suppressed it recently, and my code didn't work 😅